services/web/server/tests/unit/with_dbs/docker-compose-devel.yml

```diff
@@ -1,7 +1,8 @@
 version: "3.8"
 services:
   postgres:
-    image: "postgres:10.11@sha256:2aef165ab4f30fbb109e88959271d8b57489790ea13a77d27c02d8adb8feb20f"
+    image: "postgres:10.11@sha256:2aef165ab4f30fbb109e88959271d8b57489790ea13a77d27\
+      c02d8adb8feb20f"
```

Comment:
this is a bit weird... is that autoformatting?

Reply:
yes :-/